### PR TITLE
add new stuff on line 3

### PR DESCRIPTION
**Pull request by alicehsing**

### CODE REVIEW

**Review by clayton-knapp — COMMENTED** [review]
this looks good!

File: app.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 //make new change by coder A
+
```
Comment (joshua360x) [inline app.js]: this line is blank, I like pizza

**Review by clayton-knapp — APPROVED** [review]
approved by me!

**Review by joshua360x — CHANGES_REQUESTED** [review]
please see my comment

**Review by PeteSeeling — APPROVED** [review]

**Review by joshua360x — APPROVED** [review]

**Pull request merged**